.github/workflows/ci.yml

```diff
@@ -15,891 +15,43 @@ concurrency:
-    # - name: Coverage Report
-    #   run: ninja -C build -v coverage-xml
-    # - name: CodeCov.io
-    #   uses: codecov/codecov-action@v4
-    #   with:
-    #     file: ./build/meson-logs/coverage.xml
-    #   env:
-    #     CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-
-  icc:
-    runs-on: ubuntu-22.04
-    env:
-      CC: /home/runner/.local/bin/icx
-      CXX: /home/runner/.local/bin/icpx
-      CFLAGS: -Wall -Werror -march=native -fp-model precise
-      CXXFLAGS: -Wall -Werror -march=native -fp-model precise
-    steps:
-    - uses: actions/checkout@v4
-    - name: CPU Information
-      run: cat /proc/cpuinfo
-    - name: Install APT Dependencies
-      run: sudo apt-get install -y ninja-build python3-pip && sudo pip install meson==0.64
-    - name: Install ICC
-      run: |
-        # download the key to system keyring
-        wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB \
-        | gpg --dearmor | sudo tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
-        # add signed entry to apt sources and configure the APT client to use Intel repository:
-        echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
-        sudo apt-get update
-        sudo apt-get install -y intel-oneapi-compiler-dpcpp-cpp
-        mkdir -p ~/.local/bin/ || true
-        for exe in icx icpx; do
-          printf '#!/bin/bash\nARGS="$@"\nsource /opt/intel/oneapi/compiler/latest/env/vars.sh >/dev/null\n%s ${ARGS}\n' "${exe}" > ~/.local/bin/"${exe}"
-          chmod 0755 ~/.local/bin/"${exe}";
-        done
-    - name: Configure
-      run: meson setup build
-    - name: Build
-      run: ninja -C build -v
-    - name: Test
-      run: meson test -C build --print-errorlogs $(meson test -C build --list | grep -v emul)
-
-  msvc-arm64:
-    name: Windows MSVC-AArch64
-    runs-on: [windows-latest]
-    env:
-      CFLAGS: /WX /Z7
-      CXXFLAGS: /WX /Z7
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          submodules: recursive
-      - name: Install Meson, Ninja and coverage
-        run: |
-          python3 -m pip install --upgrade ninja meson
-          choco install opencppcoverage codecov
-      - name: Use ARM64 Developer Command Prompt
-        uses: ilammy/msvc-dev-cmd@v1
-        with:
-          arch: amd64_arm64
-      - name: Configure and Build
-        run: |
-          meson --backend=ninja build --cross-file test/arm64cl.txt
-          ninja -C build test
 
   linux-gcc-loongarch64:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
     - uses: actions/checkout@v4
     - name: CPU Information
       run: cat /proc/cpuinfo
     - name: Install APT Dependencies
       run: |
-        sudo add-apt-repository ppa:savoury1/virtualisation
         sudo apt-get install -y ninja-build ninja-build meson qemu-user-static
```

Comment:
I will try to confirm if this qemu support LSX.

Reply:
It should, https://packages.ubuntu.com/oracular/qemu-user-static is version `1:9.0.2+ds-4ubuntu3`

Checking by running the container locally I get
```
root@27baec51c897:/# qemu-loongarch64-static -cpu help
Available CPUs:
  la132
  la464
  max
root@27baec51c897:/# qemu-loongarch64-static -version
qemu-loongarch64 version 9.0.2 (Debian 1:9.0.2+ds-4ubuntu3)
Copyright (c) 2003-2024 Fabrice Bellard and the QEMU Project developers
```

Reply:
Yep, the CI passed with `-cpu la464` to qemu against `-mlsx` with gcc

https://github.com/simd-everywhere/simde/actions/runs/10809969904/job/29986199744?pr=1217